charts/jenkins/VALUES.md

```diff
@@ -0,0 +1,298 @@
+| controller.httpsKeyStore.jenkinsHttpsJksSecretName | string | `""` | Name of the secret that already has ssl keystore |
+| controller.httpsKeyStore.jenkinsKeyStoreBase64Encoded | string | `nil` | Base64 encoded Keystore content. Keystore must be converted to base64 then being pasted here |
+| controller.httpsKeyStore.password | string | `"password"` | Jenkins keystore password |
+| controller.httpsKeyStore.path | string | `"/var/jenkins_keystore"` | Path of HTTPS keystore file |
+| controller.image.pullPolicy | string | `"Always"` | Controller image pull policy |
+| controller.image.registry | string | `"docker.io"` | Controller image registry |
+| controller.image.repository | string | `"jenkins/jenkins"` | Controller image repository |
+| controller.image.tag | string | `nil` | Controller image tag override; i.e., tag: "2.440.1-jdk17" |
+| controller.image.tagLabel | string | `"jdk17"` | Controller image tag label |
+| controller.imagePullSecretName | string | `nil` | Controller image pull secret |
+| controller.ingress.annotations | object | `{}` | Ingress annotations |
+| controller.ingress.apiVersion | string | `"extensions/v1beta1"` | Ingress API version |
+| controller.ingress.enabled | bool | `false` | Enables ingress |
+| controller.ingress.hostName | string | `nil` | Ingress hostname |
+| controller.ingress.labels | object | `{}` | Ingress labels |
+| controller.ingress.path | string | `nil` | Ingress path |
+| controller.ingress.paths | list | `[]` | Override for the default Ingress paths |
+| controller.ingress.resourceRootUrl | string | `nil` | Hostname to serve assets from |
+| controller.ingress.tls | list | `[]` | Ingress TLS configuration |
+| controller.initConfigMap | string | `nil` | Name of the existing ConfigMap that contains init scripts |
+| controller.initContainerEnv | list | `[]` | Environment variables for Init Container |
+| controller.initContainerEnvFrom | list | `[]` | Environment variable sources for Init Container |
+| controller.initContainerResources | object | `{}` | Resources allocation (Requests and Limits) for Init Container |
+| controller.initScripts | object | `{}` | Map of groovy init scripts to be executed during Jenkins controller start |
+| controller.initializeOnce | bool | `false` | Initialize only on first installation. Ensures plugins do not get updated inadvertently. Requires `persistence.enabled` to be set to `true` |
+| controller.installLatestPlugins | bool | `true` | Download the minimum required version or latest version of all dependencies |
+| controller.installLatestSpecifiedPlugins | bool | `false` | Set to true to download the latest version of any plugin that is requested to have the latest version |
+| controller.installPlugins | list | `["kubernetes:4186.v1d804571d5d4","workflow-aggregator:596.v8c21c963d92d","git:5.2.1","configuration-as-code:1775.v810dc950b_514"]` | List of Jenkins plugins to install. If you don't want to install plugins, set it to `false` |
```

Comment:
the size of this is unfortunate, at least for me it adds a lot of scrolling, meaning the description field isn't visible and I have to scroll to it.

Reply:
I'll see if I can change the Markdown template generated by helm-docs, I think that should be possible